index.js

```diff
@@ -0,0 +1,65 @@
+// getting html elements
+const bookList = document.querySelector('.book_list');
+const title = document.querySelector('.title');
+const author = document.querySelector('.author');
+const addBtn = document.querySelector('.add_btn');
+
+let books = [];
+books = JSON.parse(localStorage.getItem('Awesome books')) !== null ? (books = JSON.parse(localStorage.getItem('books'))) : [];
+
+let listBooks;
+const getLocalData = () => {
+  listBooks = JSON.parse(localStorage.getItem('books'));
+};
+// show books function
+
+const showListBooks = () => {
+  bookList.innerHTML = '';
+  getLocalData();
+  listBooks.forEach((element) => {
+    const listTag = `
+<p>${element.title}</p>
+<p>${element.author}</p>
+<button type="button" class="remove" id="${element.id}">Remove</button>
+<hr>
+            `;
+    bookList.innerHTML += listTag;
+  });
+};
+//   Addding books
+
+const addBook = () => {
```

Comment:
- [ ] As the previous reviewer requested, it is impossible to add a book no matter how many times I press the "Add" button (tested both on Firefox and Chrome).  Please correct this app behavior.

## HInts

- I suspect the problem is in line 32 when you try to create the local storage for the first time.
- It works for you because you already have your local storage created and saved on your browser.  But it does not work for anyone else.
-  Try to delete your local storage and then try to create a new book, in order to reproduce the issue, so you can debug it.
- You delete your local storage by going to dev tools => Application => Local Storage =>  Delete anything that appears on that list:

![image](https://user-images.githubusercontent.com/57421823/166648352-c1d75af9-31fe-4d5b-95df-31ebd6c982f1.png)

That way, you will be able to debug the issue 💪

Reply:
@Meltrust Thank you for your comprehensive feedback.
I have fixed the problem